metricbeat/module/haproxy/info/info_integration_test.go

```diff
@@ -22,10 +22,26 @@ package info
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 	"github.com/elastic/beats/metricbeat/module/haproxy"
 )
 
+func TestFetch(t *testing.T) {
+	compose.EnsureUp(t, "haproxy")
```

Comment:
@jsoriano I assume for now this means it's only tested with 1 haproxy version or will it check multiple?

Reply:
This is only tested with one version.